cypress/integration/timeline.js

```diff
@@ -50,4 +52,43 @@ context('Timeline', () => {
 		cy.get('.menu-btn-group > .dropdown-menu > li > .grey-link').eq(17).click({force: true});
 		cy.get('.modal.show > .modal-dialog > .modal-content > .modal-footer > .standard-actions > .btn-primary').contains('Yes').click({force: true});
 	});
+
+	it('Cancel functionality for timeline', () => {
+		cy.visit('/app/doctype');
+
+		//Creating custom doctype
+		cy.insert_doc('DocType', custom_submittable_doctype, true);
+
+		cy.visit('/app/custom-submittable-doctype');
+		cy.click_listview_primary_button('Add Custom Submittable DocType');
+
+		//Adding a new entry for the created custom doctype
+		cy.fill_field('title', 'Test');
+		cy.get('.modal.show > .modal-dialog > .modal-content > .modal-footer > .standard-actions > .btn-primary').click();
+		cy.get('.modal-footer > .standard-actions > .btn-primary').contains('Submit').click();
+		cy.visit('/app/custom-submittable-doctype');
+		cy.get('.list-row > .level-left > .list-subject > .bold > .ellipsis').eq(0).click();
+
+		//To check if the submission of the documemt is visible in the timeline content
+		cy.get('.timeline-content').should('contain', 'Administrator submitted this document');
+		cy.get('.page-head > .container > .row > .col > .standard-actions > .btn-secondary').contains('Cancel').click({delay: 900});		cy.get('.modal.show > .modal-dialog > .modal-content > .modal-footer > .standard-actions > .btn-primary').contains('Yes').click();
```

Comment:
```suggestion
		cy.get('.page-head > .standard-actions > .btn-secondary').contains('Cancel').click({delay: 900});
		cy.get('.modal.show > .modal-footer > .standard-actions > .btn-primary').contains('Yes').click();
```

We can skip few classes and make a selector shorter if the target has a unique class (on the page or inside an element).